client/packages/system/src/Stock/ListView/AppBarButtons.tsx

```diff
@@ -9,18 +9,24 @@ import {
   LoadingButton,
   EnvUtils,
   Platform,
+  PlusCircleIcon,
+  ButtonWithIcon,
+  useEditModal,
 } from '@openmsupply-client/common';
 import { useStock } from '../api';
 import { stockLinesToCsv } from '../../utils';
+import { NewStockLineModal } from '../Components/NewStockLineModal';
 
 export const AppBarButtonsComponent = () => {
   const { success, error } = useNotification();
-  const t = useTranslation('distribution');
+  const t = useTranslation('inventory');
```

Comment:
Don't we pass t to stockLinesToCsv ?

Reply:
We do, but all the translations used by stockLinesToCsv are actually from the `common` translations, so the switch is fine... really would be awesome if there was a way to type these!

Reply:
Yeah i made a comment about it a few times, i actually not certain that having different files helps us